src/extensions/default/DebugCommands/main.js

```diff
@@ -230,9 +230,9 @@ define(function (require, exports, module) {
         });
     }
     
-    function toggleErrorNotification(bool) {
+    function togglePref(key, bool) {
         var val,
```

Comment:
These changes to DebugCommands/main.js are not necessary. I only refactored this code from `toggleErrorNotification()` into `togglePref()` to reduce duplication with new function, but it was moved to `LiveDevelopment/main.js` so there's no need for any changes here.